### ISSUE TITLE
Don't print non-enumerable symbols

### ISSUE THREAD

**Issue by novemberborn** [avajs/pretty-format#3]
When printing objects, only enumerable property names are printed. This seems reasonable. However, all *property symbols* are printed, whether they're enumerable or not.

As suggested by @caesarsol in https://github.com/avajs/pretty-format/pull/2#issuecomment-280309497.

**Comment by vadimdemedes** [comment]
I agree, if it doesn't affect assertion, it shouldn't be printed. They aren't `enumerable`, so it's kind of by definition that we shouldn't list them.
